src/transformers/models/gpt_neox/configuration_gpt_neox.py

```diff
@@ -99,6 +99,7 @@ def __init__(
         bos_token_id=0,
         eos_token_id=2,
         tie_word_embeddings=False,
+        gpt_j_residual=False,
```

Comment:
This name is not informative at all. Reading the code it's more a `add_residual_at_end` or something along those lines. The new parameter will also require documentation.

Reply:
I renamed it to "use_parallel_residual" and set the default value to True so that the released "gpt-neox-20b" doesn't need to change the config file.